rust/connlib/shared/src/messages/client.rs

```diff
@@ -222,6 +230,10 @@ impl PartialOrd for ResourceDescription {
 
 impl Ord for ResourceDescription {
     fn cmp(&self, other: &Self) -> std::cmp::Ordering {
+        if self.is_internet_resource() {
+            return std::cmp::Ordering::Less;
+        }
+
         (self.name(), self.id()).cmp(&(other.name(), other.id()))
     }
 }
```

Comment:
Does this ordering do anything? Shouldn't this be implemented on the type in `callbacks`?

Reply:
I guess we are sorting it first and then converting it to the other `callbacks` type. Can we change that and move the `Ord` impl to the type in `callbacks`? I think that would make more sense.